pkg/exchange/coinbase/exchange.go

```diff
@@ -120,6 +120,19 @@ func (e *Exchange) QueryAccountBalances(ctx context.Context) (types.BalanceMap,
 	return balances, nil
 }
 
+func (e *Exchange) queryAccountIDs(ctx context.Context) (map[string]string, error) {
```

Comment:
i feel like you need a queryAccounts method, and another "toAccountCurrencyIds" function to convert the accounts into the map type

Reply:
Yes, I add a similar function, `queryAccountIDsBySymbols`, instead.
Internally `queryAccountIDsBySymbols` will call `queryAccountIDs` and convert the map accordingly.

I can't use `QueryAccount` directly is that it would lost the account id in the `types.Account`.
So I have to use the raw get balance request here.